### PR TITLE
User story 17 - Item image now link to items show page

### PR DESCRIPTION
**Pull request by tylertomlinson**
**ALWAYS PULL FROM MASTER BEFORE CREATING PR**

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document? (need to decided as group what to put here)
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls)? (for reducing merge conflicts)

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Have you lint your code locally prior to submission?

### Testing:

* [x] Does your submission pass tests?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

**Test coverage** = 100%
--------------------------------------------------------------------------------
Image source on `items index page` now links to `items show page`
Test added within `items index spec`

### CODE REVIEW

File: spec/features/items/index_spec.rb [hunk 1]
```diff
@@ -23,6 +23,15 @@
       expect(page).to have_link(@dog_bone.merchant.name)
     end
 
+    it "all items images are links to item show page" do
```
Comment (Mariana-21) [inline spec/features/items/index_spec.rb:26]: Very robust test!

**Review by Mariana-21 — APPROVED** [review]
Nice way to make the image a link! and really nice test, really robust.

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <section class = "grid-item" id= 'item-<%=item.id%>'>
       <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
       <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-      <img src= <%= item.image %>>
+      <%= link_to image_tag(item.image), "/items/#{item.id}", id: "img-#{item.id}" %>
```
Comment (alex-latham) [inline app/views/items/index.html.erb:12]: Good use of rails' image_tag

**Review by alex-latham — APPROVED** [review]
Good to go

**Pull request merged**